challenge-06/challenge-06.js

```diff
@@ -58,9 +75,43 @@ Crie uma função chamada `convertToHex`, com as seguintes características:
     a frase:
     "Não temos o equivalente hexadecimal para [COR]."
 */
-// ?
+
+function convertToHex(cor){
+    switch(cor){
+        case 'azul':
+            return('O hexadecimal para a cor ' + cor + ' é 0000f9.');
```

Comment:
-1
@Feabbud, perceba que a frase de retorno é praticamente a mesma em todos os `case`. O que muda? Somente a cor.

Então, o que você pode fazer para melhorar isso? Declare uma variável no início da função, sem valor, que irá receber a cor hexadecimal dentro do `switch`, conforme a cor escolhida. Então, no final da função, você faz um único `return` com a frase toda, concatenando também a cor hexadecimal :)

Pode deixar só o return do `default`, já que a frase é diferente :)